docs/metrics.md

```diff
@@ -1,4 +1,4 @@
-<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+<!-- AUTOGENERATED BY glean_parser. DO NOT EDIT. DO NOT COMMIT. -->
```

Comment:
HAH! Of course this is wrong :) Docs should be committed :-P @brizental we both missed this!

Reply:
AAAAAA as foreseen, this change in the template would blame anyone who commited these files.

Reply:
Who would have know anyone would be YOU!

Reply:
Shame is on me adding the "DO NOT COMMIT" warning to the docs files, which actually should be commited... I'll remove myself now 😄